xla/service/gpu/fusions/fusions.cc

```diff
@@ -145,12 +145,12 @@ absl::StatusOr<std::unique_ptr<FusionInterface>> GetFusionEmitter(
     if (!opts.xla_gpu_enable_mlir_emitters()) {
       return false;
     }
-    CHECK(!check ||
```

Comment:
It seems like the check argument to the lambda is now unused after this change?

Reply:
Addressed review comment in https://github.com/openxla/xla/pull/12579/commits/437ddd8670b45fbfdb9bda892388d2189ca91ffe